src/tooling/tooling-simple-types.md

```diff
@@ -0,0 +1,232 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Tooling with Simple Types
```

Comment:
There is already a similar page explaining the setup (Hello World), and this one repeats the same workflow for each class (`Object`, `Resource`, `Node`). 

Maybe some ideas:
- Recipes should ideally focus on some more advanced use cases. I can gladly cover the whole class registration setup in the base tutorial, and your writing could complement this 🙂 
- Maybe on this page, you could focus on the `Resource` class, as it's a bit more advanced? There are also some related APIs like `load`/`save` [here](https://godot-rust.github.io/docs/gdext/master/godot/engine/index.html#functions).

Reply:
I like that idea. When I was first learning godot-rust the hello world pages were fairly confusing, so I'd be happy to expand them a bit.

I do agree though that focusing on Resource as an interesting implementation approach is a good use of the space in this recipe